Code/BasicFilters/json/LabelShapeStatisticsImageFilter.json

```diff
@@ -86,7 +86,8 @@
         }
       ],
       "custom_cast" : "sitkITKImageRegionToSTL(value)",
-      "label_map" : true
+      "label_map" : true,
+      "detaileddescription" : "Get the bounding box of a labeled object.  The bounding box is provided as a vector of size 2*ImageDimension. The first half of the vector is the starting index of the bounding box, and the second half is the size of the bounding box."
```

Comment:
Not sure if "vector" is the correct phrase, I guess depends on the programming language. I don't have anything better in mind.

Reply:
Here is the documentation for the how we represent the ImageRegion as a vector in SimpleITK: https://simpleitk.readthedocs.io/en/master/conventions.html#image-regions-as-index-and-size